src/firmware/main/io/dribbler.c

```diff
@@ -7,11 +7,11 @@
 #include <stdio.h>
 
 #include "control/control.h"
+#include "firmware/main/shared/physics.h"
 #include "io/adc.h"
 #include "io/hall.h"
 #include "io/motor.h"
 #include "io/receive.h"
```

Comment:
same as above

Reply:
Similar comment to above, out-of-scope for this PR.